modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -154,15 +154,48 @@ Displays all records that includes a valid owner. For example, it includes the r
 [NOTE]
 Limit the number of objects to retrieve through DataSense to a few objects using search specifications, otherwise retrieving metadata slows down Studio.
 
-== Log Requests and Responses [[log-requests-responses]]
+[[log-requests-responses]]
+== Log Requests and Responses 
 
-To log requests and responses when using the connector, configure the `log4j.xml` configuration file for the Mule app as follows:
+To log requests and responses when using the connector, add the following line to the `Loggers` element of the `log4j2.xml` configuration file for the Mule app:
 
 [source,xml,linenums]
 ----
 <AsyncLogger name="org.mule.modules.siebel.internal.service"
    level="DEBUG"/>
 ----
+
+For example:
+
+[source,xml,linenums]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+	<Appenders>
+	    <Console name="Console" target="SYSTEM_OUT">
+	    	<PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+		</Console>
+	</Appenders>
+	<Loggers>
+		<Logger name="org.mule.modules.siebel.internal.service" level="debug" additivity="false">
+			<appender-ref ref="Console" level="debug"/>
+		</Logger>
+	</Loggers>
+</Configuration>
+----
+
+You can view the app log  as follows:
+
+* If you’re running the app from the Anypoint Platform, the output is visible in the Anypoint Studio console window.
+
+* If you’re using Mule from the command line to run the app, the app log is visible in your OS console.
+
+You can also view the app log in this default location, unless the log file path was customized in the app’s log file (`log4j2.xml`):
+
+`MULE_HOME/logs/<app-name>.log`
+
+For more information about the app log, see xref:mule-runtime::logging-in-mule.adoc[Configuring Logging].
+
 == Use Case: Studio
 
 The Siebel Connector is an operation-based connector,
```

Comment:
Hidden TEXT

which means that when you add the connector to your flow, you must specify an operation to perform.

=== Example Use Cases
The following are common use cases for the Siebel v4.0.0 connector:
. Create a business component (Action, Account, and so on).
.. Create a new Mule project in Anypoint Studio.
.. Add a suitable Mule inbound endpoint, such as the HTTP listener, to begin the flow.
.. Drag the create business component operation from the Siebel JDB palette onto the canvas and then select it to open the properties editor.
.. Configure the operation's parameters:
+
image::siebel/siebel-studio-create.png[Configure the operation's parameters]
+
.. Drag the Transform Message connector between the HTTP Listener and the Siebel connector on the flow.
.. Configure the input payload with the desired fields:
+
image::siebel/siebel-studio-transform-create.png[Configure the input payload]
+

NOTE THAT this doc uses both "HTTP Listener" and "HTTP listener". Standardize to the correct presentation.

Reply:
HIDDEN TEXT

. Execute a business service (Create Contact, Create Account, and so on).
+
.. Create a new Mule project in Anypoint Studio.
.. Add a suitable Mule inbound endpoint, such as the HTTP listener, to begin the flow.   
NOTE Listener vs. listener; search for and standardize GLOBALLY
.. Drag the execute business service operation from the Siebel JDB palette onto the canvas and then select it to open the properties editor.
.. Configure the operation's parameters:
+
image::siebel/siebel-execute.png[Configure the operation parameters]
+
.. Drag the Transform Message connector between the HTTP Listener and the Siebel connector on the flow.
.. Configure the input payload with the desired fields:
+
image::siebel/siebel-transform-execute.png[Configure the input payload]
+
. Execute Siebel adapter (creating an integration object)
+
.. Create a new Mule project in Anypoint Studio.
.. Add a suitable Mule inbound endpoint, such as the HTTP listener, to begin the flow.
.. Drag the create business component operation from the Siebel JDB palette onto the canvas and then select it to open the properties editor.
.. Configure the operation's parameters:

Reply:
MORE HIDDEN TEXT

+
image::siebel/siebel-integration.png[Configure the operation parameters]
+
.. Drag the Transform Message connector between the HTTP Listener and the Siebel connector on the flow.
.. Configure the input payload with the desired fields:
+
image::siebel/siebel-transform-integration.png[Configure the input payload]

== Use Case: XML
CAN WE MAKE THIS A COMPLETE SENTENCE?
XML snippet from a project using the Siebel Connector:
[source,xml,linenums]
----
<?xml version="1.0" encoding="UTF-8"?>
<mule xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xmlns:siebel="http://www.mulesoft.org/schema/mule/siebel"
      xmlns:http="http://www.mulesoft.org/schema/mule/http"
      xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd